First batch of adding descriptions to the documentation (#14825)
netdata/netdata: docs/cloud/manage/plans.md

# Netdata Plans

> This page will guide you through Netdata's Plans, and what each of Community, Pro, Business and Enterprise provide.

Reviewer: ```suggestion
This page will guide you through the differences between the Community, Pro, Business and Enterprise plans.
```
